doc/dev/transforms.rst

```diff
@@ -305,12 +473,8 @@ changes (also called the insertion point). Let's have look at some of these elem
 
   .. code-block:: cpp
 
-      rewriter.updateRootInPlace(op, [&] {
```

Comment:
I think it is necessary when modifying op inplace

Reply:
Ahh, they changed the name https://github.com/llvm/llvm-project/pull/78260